certified-connectors/agilite/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "apiServerUrl": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "required": "true",
+            "tabIndex": 2
+          },
+          "description": "Specify Agilit-e API Server URL. (Use 'https://api.agilite.io' for Agilit-e Cloud)",
+          "displayName": "Specify Agilit-e API Server URL. (Use 'https://api.agilite.io' for Agilit-e Cloud)",
+          "tooltip": "Specify Agilit-e API Server URL. (Use 'https://api.agilite.io' for Agilit-e Cloud)"
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "constraints": {
+            "clearText": false,
+            "required": "true",
+            "tabIndex": 3
+          },
+          "description": "API Key (Generated in 'https://portal.agilite.io' under Administration)",
+          "displayName": "API Key (Generated in 'https://portal.agilite.io' under Administration)",
+          "tooltip": "API Key (Generated in 'https://portal.agilite.io' under Administration)"
+        }
+      }
+    },
+    "iconBrandColor": "#DA262E"
+  }

```

Comment:
@agilitehub Can you please add `policyTemplateInstances` section in apiProperties.json:

```
"policyTemplateInstances": [
      {
        "parameters": {
          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('apiServerUrl')"
        },
        "templateId": "dynamichosturl",
        "title": "Specify Agilit-e API Server URL. (Use 'https://api.agilite.io' for Agilit-e Cloud)"
      }
    ]
```

Reply:
@agilitehub Checking in on the above